src/index.tsx

```diff
@@ -1,180 +1,4 @@
-}
-
-const renderApp = (state) => {
-
-    const action = (stateHandler) => (...args) => {
-        const newState = stateHandler(state, ...args)
-        renderApp(newState);
-    }
-
-    window.onresize = () => {
-        renderApp(state);
-    }
-
-    const statefulController = StatefulController(renderApp, state);
-    const statefulControllerByProperty = StatefulControllerByProperty(statefulController);
-    const castleRiskController = statefulControllerByProperty('castleRisk', CastleRiskInitialState);
-
-    const siteProps = {
-        ...state, 
-        CastleRisk: castleRiskController(CastleRisk),
-        addNewOrganization: action(addNewOrganization),
-        navTo: action(navTo),
-        navigateToSimpleModel: action(navigateToSimpleModel),
-        onCelebrationXClicked: action(onCelebrationXClicked),
-        onFacebookLoginComplete: action(onFacebookLoginComplete),
-        onFacebookLoginFailure: action(identity),
-        onHamburgerClicked: action(onHamburgerClicked),
-        onLogoutRequested: action(onLogoutRequested),
-        onSearchBarValueChange: action(onSearchBarValueChange),
-        onSideNavCollapseRequested: action(onSideNavCollapseRequested),
-        onSideNavExpandRequested: action(onSideNavExpandRequested),
-        updateNewOrgLogo: action(updateNewOrgLogo),
-        updateNewOrgMission: action(updateNewOrgMission),
-        updateNewOrgTitle: action(updateNewOrgTitle),
-        updateNewOrgUrl: action(updateNewOrgUrl),
-    }
-
-    ReactDOM.render(
-        <Site {...siteProps} />,
-        document.getElementById("site")
-    );
-}
-
-const facebookUserSessionCookieKey = 'facebookUserSession';
-const deleteSessionCookie = () => Cookies.remove(facebookUserSessionCookieKey);
-const facebookUserSessionString = Cookies.get(facebookUserSessionCookieKey);
-const facebookUserSession = facebookUserSessionString && JSON.parse(facebookUserSessionString);
-const isLocalhost = () => window.location.hostname == "localhost";
-const developers = ['tom@tommysullivan.me','mrcorn123@yahoo.com','tastulae@mail.usf.edu',"aashreya.isforever@gmail.com", 'patmetsch@roadrunner.com', 'nishijain2512@gmail.com']
-const isDeveloper = facebookUserSession => facebookUserSession && developers.includes(facebookUserSession.email);
-const inDevMode = isLocalhost() || isDeveloper(facebookUserSession);
-
-const initialState = {
-    facebookUserSession, 
-    inDevMode,
-    isSideNavExpanded: false,
-    isSideNavOpen: false,
-    newOrgLogo: [],
-    newOrgMission: '',
-    newOrgTitle: '',
-    newOrgUrl: '',
-    organizations: defaultOrganizations,
-    searchBarSuggestions: [],
-    searchBarValue: '',
-    shouldShowCelebration: false,
-    shouldShowSearch: true,
-    showSimpleModel: false, 
-    url: Url(window.location.href),
-    volunteeringSearchBarSuggestions: [],
-    volunteeringSearchBarValue: '',
-}
-
-renderApp(initialState);
\ No newline at end of file
+renderApp(initialState());
```

Comment:
ooooo single line entrypoint file :O

Reply:
yes! would like to rename it something more descriptive but didnt feel like updating all the places like webpackconfig etc. another day :)